### PR TITLE
Reduce crypt3.sh parameters

### PR DESCRIPTION
**Pull request by jeancsil**
Using _which_ command to auto find _openssl_ executable.
Changed from tabs to spaces.
Some comments were removed..

### CODE REVIEW

File: scripts/crypt3.sh [hunk 1]
```diff
@@ -2,115 +2,112 @@
...
-command=${2}
-shred=${3}
-key=${4}
-filePath=${5}
-skipChecks=${6}
+if [[ $# != 4 ]]; then
+    usage
+fi
+
+opensslexe=`which openssl`
```
Comment (jeancsil) [inline scripts/crypt3.sh:17]: I'll also check if the openssl is installed.

File: scripts/crypt3.sh [hunk 2]
```diff
@@ -2,115 +2,112 @@
...
+command=${1}
+shred=${2}
+key=${3}
+filePath=${4}
+skipChecks=${5}
```
Comment (jeancsil) [inline scripts/crypt3.sh:22]: This parameters doesn't exist, right?

Comment (stevedekorte) [inline scripts/crypt3.sh:22]: What do you mean? All those parameters are passed in my the Crypt application.

Comment (jeancsil) [inline scripts/crypt3.sh:22]: Sorry, I meant the skipChecks parameter.. before (the parameter number 6) was not in the usage documentation.. now is the number 5 but still not existent.. right?

Comment (stevedekorte) [inline scripts/crypt3.sh:22]: Ah, yeah I don't think skipChecks was used.

**Comment by stevedekorte** [comment]
Thanks Jean.

**Pull request merged**